### ISSUE TITLE
Switches on Fabric should be Accessible and Focusable by Default

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#11857]
### Problem Description

The Switch control on Fabric is currently not in the UIA tree and not focusable by default. On Paper, the Switch control was in the UIA tree and focusable by default. The behavior of the Switch control should be the same for both architectures. 

### Steps To Reproduce

1. Run playground-composition.sln.
2. Add a Switch control to a sample page.
3. Observe that the control cannot be tabbed to by default. 
4. Open AccessibilityInsights and see that the switch control is not in the UIA tree. 

### Expected Results

Switch control should be able to be tabbed to by default. Control should be seen in the UIA tree as a button control. 

### CLI version

npx react-native -v

### Environment

```markdown
npx react-native info
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None
